test/git/__snapshots__/Branch.test.ts.snap

```diff
@@ -1,5 +1,5 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
+// Vitest Snapshot v1
 
-exports[`Branch getName ignores unknown template data 1`] = `"{does_not_exist}"`;
+exports[`Branch > getName > ignores unknown template data 1`] = `"{does_not_exist}"`;
 
-exports[`Branch getName replaces the placeholders with the issue data 1`] = `"fotingo-2484.c.Legacy"`;
+exports[`Branch > getName > replaces the placeholders with the issue data 1`] = `"fotingo-2484.c.Legacy"`;
```

Comment:
Looks like vitest handles nested describe and test blocks a little differently, adding `>` in between.